Slurper.js

```diff
@@ -0,0 +1,88 @@
+const { EventEmitter } = require('events')
+const { StringUtil } = require('./lib/Util')
+
+module.exports = class Slurper extends EventEmitter {
+
+    static BindEvent( key, event, el, els ) {
+        const eventEls = el ? [ el ] : Array.isArray( els[ key ] ) ? els[ key ] : [ els[ key ] ],
+           name = this.getEventMethodName( key, event )
```

Comment:
A static class should not use `this`

Reply:
That is not correct: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes/static#Calling_static_methods. Static methods in the same class can call each other with `this` all they want. It would be a mess getting all of this to work otherwise.

Reply:
I know it works, but I don't like the pattern.  Breaking it out into its own file is just confusing when you have `this` referencing things in another file.  Yes, I did this on the backend and with TemplateContext, I wish I did not do so.